Ft mobile skeleton

felixodette opened the pull request:
Made a mobile layout skeleton for the project. Included the following files:

Made a mobile layout skeleton for the project. Included the following files:
- index.html
- style.css
- .gitignore
- .github/workflows/linters.yml
- img/...
- README.md

felixodette committed:
Project setup

felixodette committed:
Project setup

felixodette committed:
Mobile layout skeleton

felixodette committed:
CSS changes for linter

felixodette committed:
Fix stylelint errors

felixodette committed:
Stylelint error fixes

felixodette committed:
Webhint href fix

felixodette committed:
Webhint lint fixes

felixodette committed:
Edited README.md

felixodette committed:
Removed .idea folder

felixodette committed:
Edited headings in README.md

felixodette committed:
Headers  in README.md

AmaduKamara commented on index.html:
```diff
@@ -0,0 +1,40 @@
+                <div class="rectangle"></div>
+            </div>
+        </nav>
+    </header>
```
# Required Changes ♻️

- [x] Good job you using the header tag for this section. It would be nice you move it out of the `<div class="container">` and let it be the first child of the body.
- [x] Please replace the logo div with an `<a>` tag in order for it to be a link and move it outside of the nav tag.
- [x] Also, it would be nice the header section has a background image in your style and is not placed directly as an `<img>` element. Please remove those images and style the header to have a background-mage of that image.
- [x] Please match the design as given in the Figma template you used.
- Figma Design:

![image](https://user-images.githubusercontent.com/50941074/158785571-50fa1cec-78cc-41ae-9c9a-329c7cecab46.png)

- Your Design
![image](https://user-images.githubusercontent.com/50941074/158785728-ec2a5603-5d51-465d-98b6-6bc1e8acbdc4.png)

AmaduKamara commented on index.html:
```diff
@@ -0,0 +1,40 @@
+        </nav>
+    </header>
+    <main>
+        <div class="name">
+            <h1>Felix Odete</h1>
+        </div>
+        <div class="about-me">
+            <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+                If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+        </div>
+        <div class="rectangle-button">
+            <h1>Start collaboration</h1>
+        </div>
```
# Required Changes ♻️ 
- [ ] Please place these in a `<section>` tag as the container of the headline section. 👍. Please make all your sections of the site be in a `<section>` tag

AmaduKamara commented on css/style.css:
```diff
@@ -0,0 +1,97 @@
+  background-color: #e5e5e5;
+}
+
+.container {
+  width: 325px;
+  justify-content: center;
+  border: 1px solid blue;
+}
```
- [ ] Please remove the purple border placed in the container.
Here is the Figma Design you should replicate
![image](https://user-images.githubusercontent.com/50941074/158786258-7838dcdd-131f-4989-8527-3b7e5f64b92a.png)

AmaduKamara reviewed (changes requested):
# STATUS: REQUIRED CHANGES ♻️♻️♻️
Hi @felixodette , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![ClappingLeonardoDicaprioGIF](https://user-images.githubusercontent.com/50941074/158786679-f66be729-b617-486c-b62c-cda467d885c4.gif)

## Highlights
✔️ No Linter errors
✔️ Professional README
✔️ Nice descriptive PR
✔️ Good commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

felixodette committed:
Implemented Code Reviewer's changes.

felixodette committed:
Implemented Code Reviewer's changes.

felixodette commented:
Thanks for the review! I've done the following changes:

-  Moved the header out of the <div class="container"> and let it be the first child of the body.
- Replaced the logo div with a <a> tag in order for it to be a link and moved it outside of the nav tag.
- Remove those images and style the header to background images styled in CSS.
- Matched the design as given in the Figma template I used.

felixodette commented:
Thanks for the review! I've done the following changes:

-  Moved the header out of the <div class="container"> and let it be the first child of the body.
- Replaced the logo div with a <a> tag in order for it to be a link and moved it outside of the nav tag.
- Remove those images and style the header to background images styled in CSS.
- Matched the design as given in the Figma template I used.

thecodechaser commented on index.html:
```diff
@@ -0,0 +1,46 @@
+                        If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+                </div>
+                <div class="rectangle-button">
+                    <h1>Start collaboration</h1>
```
- [ ] [optional] Please replace this `<h1>` heading with other headings example H2, H3, etc. A webpage should only have one `<h1>` that describes what a webpage is about.

thecodechaser reviewed (approved):
## STATUS: APPROVED 🟢

## Hi @felixodette👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️

## Optional suggestions
- [ ] Please check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
